packages/core/useFetch/index.ts

```diff
@@ -55,6 +55,7 @@ interface UseFetchReturnBase<T> {
 
 type DataType = 'text' | 'json' | 'blob' | 'arrayBuffer' | 'formData'
 type PayloadType = 'text' | 'json' | 'formData'
+type Context = { url: string; options: RequestInit }
```

Comment:
```suggestion
type Context = { url: string; options: RequestInit, cancel: () => void }
```

Ideally, we sure use an interface with some jsdocs

Reply:
Do we want to keep it as `Context` or rename it to something more descriptive like `BeforeFetchContext`?

Reply:
Sure, `BeforeFetchContext` sounds like a good idea 👍